Mathlib/CategoryTheory/Galois/Prorepresentability.lean

```diff
@@ -177,6 +186,232 @@ noncomputable def autGaloisSystem : PointedGaloisObject F ⥤ GroupCat.{u₂} wh
     ext (σ : Aut A.obj)
     simp
 
+/-- `autGaloisSystem` but lifted to a bigger universe. This is needed to compute its limit. -/
+noncomputable def autGaloisSystem' : PointedGaloisObject F ⥤ GroupCat.{max u₁ u₂} :=
+  autGaloisSystem F ⋙ GroupCat.uliftFunctor.{u₂, u₁}
+
+@[simp]
+lemma autGaloisSystem'_map {A B : PointedGaloisObject F} (f : A ⟶ B) (φ : Aut (A : C)) :
+    ((autGaloisSystem' F).map f) ⟨φ⟩ = ⟨autMapHom f.val φ⟩ :=
+  rfl
+
+/-- The limit of `autGaloisSystem` computed in `GroupCat.{max u₁ u₂}`. -/
+noncomputable def autGalois : GroupCat.{max u₁ u₂} := limit (autGaloisSystem' F)
```

Comment:
In order to define this limit, instead of ulifting+using the limit API, what would you think of using the `Functor.sections` API? (regardless of universes, this is a well defined type, and we have an instance `Group (F ⋙ forget GroupCat.{u}).sections` which is defined in `Algebra.Category.GroupCat.Limits`)

Reply:
I pushed an attempt. Do you think it's better this way?

Reply:
Yes, it looks better to me!